app/controllers/api/households_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module Api
+  # Exposes the Household data
+  class HouseholdsController < Api::BaseController
+    before_action :set_household, only: %i[show update delete]
+    # GET /households
+    # def index
+    #   # Should get household by current user_id/member_id
+    #   @household = Household.joins(:members)
+    #   .where(members: { id: current_user.member.id })
+    #   render json: @household
+    # end
+
+    # GET /households/1
+    def show
+      if @household
+        render json: @household
+      else
+        render json: {}, status: :not_found
+      end
+    end
+
+    # POST /households
+    def create
+      @household = Household.new(household_params)
+      set_added_by
+      if @household.save
+        render json: @household
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    # PUT /households/1
+    def update
+      if @household.update(household_params)
+        render json: @household
+      else
+        render json: @household.errors, status: :unprocessable_entity
+      end
+    end
+
+    # DELETE /households/1
+    def delete
```

Comment:
Good question! Historically Hard deletes have terrified us and we have only implemented soft deletes. I would say let's just keep it safe and enable soft deletes (some kind of status code?). If we want to build hard deletes that would take a bit more of front end design and thinking through the business rules so we can defer that capability for this round of updates.

Reply:
@csteele86 @davidpickering 

Since we will be moving towards soft-deletes, it seems like we will need to add deleted_at and deleted_by to each of our entities on the ERD. Is this correct?

Reply:
I like that approach @chahn010! Make them nullable fields too.

Reply:
Sounds great 👍